census/census.go

```diff
@@ -61,6 +62,14 @@ func handle(w http.ResponseWriter, req *http.Request, cfg *config.Config, c cach
 			return availableItems[i].Topic < availableItems[j].Topic
 		})
 
+		availableTopicIDs := make([]string, 0, len(availableItems))
+		for _, availableItemID := range availableItems {
+			availableTopicIDs = append(availableTopicIDs, availableItemID.ID)
+		}
+
+		availableIDs := strings.Join(availableTopicIDs, ",")
+		availableIDs = fmt.Sprintf("/search?topics=%s&filter=datasets", availableIDs)
```

Comment:
Since we are passing `availableItems` to the mapper, could we move this logic there instead of adding an extra parameter to the `Census` function?